skedge-backend/index.js

```diff
@@ -141,14 +158,16 @@ app.post('/users/login', (req, res, next) => {
                     message: "Authorization failed!"
                 });
             }
-            const token = jwt.sign({username: fetchedUser.username, userId: fetchedUser._id},
-                "secret_this_should_be_longer",
+            const authToken = jwt.sign({username: fetchedUser.username, userId: fetchedUser._id},
+                rsa_encryption.getRsaPrivateKey(), // Using private key as the HMAC key cuz why not
```

Comment:
I don't think the words "cuz why not" ever belong in a comment
You could write that it's arbitrary but the wording isn't super professional